stable/hazelcast-enterprise/values.yaml

```
@@ -437,5 +437,10 @@ mancenter:
     # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
     failureThreshold: 3
 
+  # Cluster config creation will create the connection to the Hazelcast cluster based on the yaml.hazelcast-client
+  clusterConfigCreation: 
+    # enabled is a flag to used to enable cluster config creation
+    enabled: true
```

`clusterConfigCreation` is a little bit longer, I think. Also we should follow the same convention with other parameters like `service` , `serviceAccount` and `rbac`:
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L146
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L172
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L167
```suggestion
  clusterConfig:
    create: true
```